Microsoft.DotNet.Interactive/Commands/NugetRestoreDirective.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Interactive.Commands
+{
+    public class NugetRestoreDirective : KernelCommandBase
```

Comment:
Command naming is typically imperative, e.g. `RestoreNugetPackage`.

Reply:
K.